test/e2e/storage/vsphere_volume_vsan_policy.go

```diff
@@ -72,7 +85,7 @@ const (
    8. Delete PVC, PV and Storage Class
 */
 
-var _ = framework.KubeDescribe("VSAN policy support for dynamic provisioning [Volume]", func() {
+var _ = framework.KubeDescribe("Storage policy support for dynamic provisioning [Volume]", func() {
```

Comment:
If this is a vSphere specific test suite then "vsphere" should appear in the description text.